keel-ec2-plugin/keel-ec2-plugin.gradle.kts

```diff
@@ -22,10 +22,20 @@ dependencies {
   testImplementation(project(":keel-test"))
   testImplementation(project(":keel-retrofit-test-support"))
   testImplementation(project(":keel-spring-test-support"))
+  testImplementation("com.netflix.spinnaker.kork:kork-plugins")
   testImplementation("io.strikt:strikt-jackson")
   testImplementation("dev.minutest:minutest")
   testImplementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml")
   testImplementation("org.funktionale:funktionale-partials")
   testImplementation("org.apache.commons:commons-lang3")
   testImplementation("org.junit.jupiter:junit-jupiter-params")
+
+  // the following are needed to use keel's real(-ish) Spring configuration
+  testImplementation(project(":keel-web")) {
+    // avoid circular dependency
+    exclude(module = "keel-ec2-plugin")
+  }
+  testImplementation("org.testcontainers:mysql:${property("testContainersVersion")}")
+  testImplementation(project(":keel-sql"))
+  testImplementation("org.liquibase:liquibase-core")
```

Comment:
It does feel like it would be easier to move the one test to `keel-web`

Reply:
In hindsight, yes it does. I'll try to tackle that in a separate PR.